haiku-apps/eventual/eventual-1.0.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A time management system"
+DESCRIPTION="Eventual is a personal time management system for Haiku. \
+Intended to be a full-featured organizer. "
```

Comment:
```diff
 DESCRIPTION="Eventual is a personal time management system for Haiku. \
-Intended to be a full-featured organizer. "
+It is intended to be a full-featured organizer."
```
You can also join the two sentences together (in a single sentence) if you prefer.